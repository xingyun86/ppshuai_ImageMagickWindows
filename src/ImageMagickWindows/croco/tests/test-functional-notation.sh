#!/bin/sh

. global-test-vars.sh

$CSSLINT $TEST_INPUTS_DIR/functional-notation.css
