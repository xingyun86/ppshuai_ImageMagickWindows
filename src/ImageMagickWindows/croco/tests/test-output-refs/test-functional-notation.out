foo {
  bar : attr(width, length, calc(70% -5px))
}

foo {
  bar : attr(x, y)
}
